lib/devise_google_authenticator.rb

```diff
@@ -19,7 +19,8 @@ module Devise # :nodoc:
   @@ga_remember_optional = false
 
   mattr_accessor :ga_appname
-  @@ga_appname = Rails.application.class.parent_name
+  # Rails 6 uses `module_parent_name`, earlier versions use `parent_name`
+  @@ga_appname = Rails.application.class.respond_to?(:module_parent_name) ? Rails.application.class.module_parent_name : Rails.application.class.parent_name
```

Comment:
Possible alternative (no preference from me)

```suggestion
  @@ga_appname =
    if Rails::VERSION::MAJOR >= 6
      Rails.application.class.module_parent_name
    else
      Rails.application.class.parent_name
    end
```

Reply:
Thanks, i like your version better.